Topics/Tech_Stacks/PrismaORM.md

```diff
@@ -0,0 +1,148 @@
+# PrismaORM
+
+## Table of Contents
+- [PrismaORM](#prismaorm)
+  - [Table of Contents](#table-of-contents)
+  - [Introduction](#introduction)
+  - [Installing Prisma](#installing-prisma)
+  - [Schema and Models](#schema-and-models)
+    - [Database Introspection](#database-introspection)
+  - [Migrations](#migrations)
+  - [Using Prisma](#using-prisma)
+
+## Introduction 
+
+While SQL is an essential skill for any serious backend developer, in many projects one might be inclined to disregard the humble SQL driver for an ORM, or Object Relational Mapper. Using relational database terms, ORMs essentially treat database tables as types, and rows in tables as individual objects so that they're easily accessed by developers, who no longer have to write queries themselves. Instead, the ORM package itself generates and run the queries when the developer calls ORM functions. This approach comes with several advantages, such as safety, simplicity, and database management system agnosticism, however be warned that certain advanced database features may be unavailable and for highly complex/specific queries you're still better off writing them yourself.
+
+PrismaORM is an ORM for JavaScript and TypeScript meant to streamline backline development with a suite of convenient features. In this guide we'll go through the basics of setting up Prisma and using it.
+
+## Installing Prisma 
+
+You install Prisma like any npm package, in your (preferably) TypeScript project run
+```
+npm install prisma --save-dev
+```
+To install the package. Then run 
+```
```

Comment:
I'm a little confused as to what this command does, could you explain what the purpose is?